.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,20 +13,20 @@
 -->
 
 This PR contains:
-
 - [ ] bugfix
 - [ ] feature
-- [ ] refactor
-- [ ] tests
+- [ ] refactoring
```

Comment:
I believe "refactor" is correct here, as the verb describes the state. The past form is "refactored," while the present participle "refactoring" describes something currently in process, the action of undertaking a refactor. While a PR could be considered in process, it's supposed to be a finished work ready for review and merging. Don't be to be pedantic about it, I just see these things and they stand out to me :smile:

Reply:
_Only_ mentioning because English is fun (seriously, I get a kick out of this stuff). bugfix, feature, and refactor are all part of a class of words called "gerunds," or verbs that are used as nouns.